drones/common/drone_instruction.py

```diff
@@ -0,0 +1,42 @@
+"""Drone instructions definitions
+"""
+
+
+def command() -> str:
```

Comment:
Is there a specific reason why these are declared as functions instead of variables e.g. `COMMAND = "Command"`?

Reply:
There are few instructions which are not parameterized, and it makes sense to make them variables - like this `COMMAND` you mentioned. Most of them, however, do have parameters and I wanted to create some sort of safety layer to avoid creation on literal strings, that's why I have chosen to make functions. This strings will be transmitted to drone and it is necessary for them to have specific format. I believe that is the best way to encapsulate them, so I kept the unparameterized ones as functions too, to keep the things unified.